src/tasks/mt.py

```diff
@@ -66,13 +66,11 @@ def load_data(self, path):
                 row = row.strip().split('\t')
                 if len(row) < 2 or not row[0] or not row[1]:
                     continue
-                src_sent = process_sentence(row[0], self.max_seq_len)
-                # target sentence sos_tok, eos_tok need to match Seq2SeqDecoder class
-                tgt_sent = process_sentence(
-                    row[1], self.max_seq_len,
-                    sos_tok=allennlp_util.START_SYMBOL,
-                    eos_tok=allennlp_util.END_SYMBOL,
```

Comment:
I assume these are set as defaults already?

Reply:
yea, and we removed these as args in the `process_sentence` function